package.json

```diff
@@ -1,37 +1,64 @@
 {
   "name": "svg-packer",
+  "type": "module",
   "version": "0.0.3",
+  "packageManager": "pnpm@10.11.0",
   "description": "Pack SVGs to Icon Fonts - In Browser!",
-  "main": "index.js",
-  "jsdelivr": "dist/index.browser.js",
-  "unpkg": "dist/index.browser.js",
-  "scripts": {
-    "build": "rimraf dist && mkdirp dist && browserify index.js --im -o dist/index.browser.js",
-    "prepublish": "yarn build"
+  "author": "Anthony Fu <anthonyfu117@hotmail.com>",
+  "license": "MIT",
+  "funding": "https://github.com/sponsors/antfu",
+  "homepage": "https://github.com/antfu/svg-packer",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/antfu/svg-packer"
   },
+  "bugs": "https://github.com/antfu/svg-packer/issues",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "main": "dist/index.js",
```

Comment:
should we change main and module? Even using a bundler (Vite) will require some transformation, check the vite.config.ts, whitout the polyfills you cannot use it.

Reply:
check https://github.com/antfu/svg-packer/pull/3#discussion_r2101329038